engine/src/main/java/org/hibernate/search/bridge/builtin/impl/NullEncodingTwoWayFieldBridge.java

```diff
@@ -58,6 +59,7 @@ public String objectToString(Object object) {
 		}
 	}
 
+	@Override
 	public TwoWayFieldBridge unwrap() {
```

Comment:
The "unwrap" method is typically implemented by letting the method return some internal component which matches the type of the parameter. That's for example the contract on `EntityManager`.

An example implementation is here:
 - org.hibernate.search.jpa.impl.FullTextEntityManagerImpl.unwrap(Class<T>)

Just wondering.. did you consider that? Did it not fit?

Your solution doesn't look bad, it's just that given the specific name I would expect it to follow the well known pattern.